variables.tf

```diff
@@ -409,3 +409,36 @@ variable "cbr_rules" {
   description = "The list of context-based restriction rules to create."
   default     = []
 }
+
+##############################################################
+# Ingress Secrets Manager Integration
+##############################################################
+
+variable "enable_secrets_manager_integration" {
+  type        = bool
+  description = "Integrate with IBM Cloud Secrets Manager so you can centrally manage Ingress subdomain certificates and other secrets. [Learn more](https://cloud.ibm.com/docs/containers?topic=containers-secrets-mgr)"
+  default     = false
+  nullable    = false
+  validation {
+    condition     = var.enable_secrets_manager_integration ? var.existing_secrets_manager_instance_crn != null : true
+    error_message = "'existing_secrets_manager_instance_crn' should be provided if setting 'enable_secrets_manager_integration' to true."
+  }
+}
+
+variable "existing_secrets_manager_instance_crn" {
+  type        = string
+  description = "CRN of the Secrets Manager instance where Ingress certificate secrets are stored. If 'enable_secrets_manager_integration' is set to true then this value is required."
+  default     = null
+}
+
+variable "secrets_manager_secret_group_id" {
+  type        = string
+  description = "Secret group ID where Ingress secrets are stored in the Secrets Manager instance. If 'enable_secrets_manager_integration' is set to true and 'secrets_manager_secret_group_id' is not provided, a new group will be created with the same name as cluster_id."
```

Comment:
Description is wrong - we are only creating the secret group in the DA. Maybe we should include it in the module too?